pkg/scheduler/core/generic_scheduler.go

```diff
@@ -1172,9 +1165,10 @@ func selectVictimsOnNode(
 }
 
 // unresolvablePredicateExists checks whether failedPredicates has unresolvable predicate.
-func unresolvablePredicateExists(failedPredicates []predicates.PredicateFailureReason) bool {
-	for _, failedPredicate := range failedPredicates {
-		if _, ok := unresolvablePredicateFailureErrors[failedPredicate]; ok {
```

Comment:
I believe we should return `predicateName` of `PredicateFailureError` as `Status.Message` in `podFitsOnNode`, so that we can keep this function. We can get the description part of `PredicateFailureError` when showing error to users.

Reply:
I updated the key of `unresolvablePredicateFailureErrors` which could help us to keep this function, what do you think with this?